neural_speed/__init__.py

```diff
@@ -21,6 +21,7 @@
 from transformers import AutoConfig, AutoTokenizer
 
 model_maps = {"gpt_neox": "gptneox", "gpt_bigcode": "starcoder"}
+max_request_num_default = 8
```

Comment:
source?

Reply:
Just an experimental value in SPR for gpt-j. Too large will affect `model mem scratch`. It will change if the use inputs bs is larger than `8`. No more ideas on how to set a more reasonable value. It may be changed if we have a better way. Any suggestions?

Reply:
make it configurable (with a default value), we surely will not stick to SPR forever